apps/server/src/modules/board/controller/mapper/card-response.mapper.ts

```diff
@@ -0,0 +1,21 @@
+import { Card } from '@shared/domain';
```

Comment:
For this mapper it exists no tests.

Reply:
do we need tests for static classes/methods?